src/Components/TeleIcu/Patient/Waveform.tsx

```diff
@@ -24,13 +24,13 @@ export default function Waveform(props: {
   classes?: string;
   defaultSpace?: boolean;
 }) {
+  const viewable = 400;
```

Comment:
This should be a const VIEWABLE outside of the component. There's no reason for it to be inside the component